sdcm/cluster.py

```diff
@@ -2207,25 +2207,25 @@ def config_scylla_manager(self, mgmt_port, db_hosts):
         else:
             self.remoter.run('sudo systemctl restart scylla-manager.service')
 
-    def start_scylla_server(self, verify_up=True, verify_down=False, timeout=300):
+    def start_scylla_server(self, verify_up=True, verify_down=False, timeout=300, verify_up_timeout=500):
```

Comment:
I think default of verify_up_timeout should remain 300 as it used to be

Reply:
Ok